adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -283,6 +283,9 @@ private AuthenticationConstants() { }
 
         /** String for request id returned from Evo. **/
         public static final String REQUEST_ID_HEADER = "x-ms-request-id";
+
+        /** Integer for token expiration buffer. see AuthenticationSettings.mExpirationBuffer */
+        public static final String EXPIRATION_BUFFER = "aad.expiration.buffer";
```

Comment:
This should be in the Broker subclass, any reason you put in the AAD subclass?

Reply:
'expiration buffer' is a concept not only related with broker, but also available for the case without broker, so I didn't put it into the 'Broker' subclass. Maybe 'AAD' is not the best place, do you have any suggestion? thanks